src/sass/messaging/partials/_messaging-app.scss

```diff
@@ -14,10 +14,41 @@
   }
 }
 
+// Override square bullets and excessive padding
+// when only using .usa-sidenav-sub_list.
+.messaging-folder-subnav {
+  list-style: none;
+  padding: 0;
+}
+
+.messaging-folder-nav-link {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.messaging-my-folders {
+  i {
+    float: right;
+    line-height: 2rem;
+  }
+}
+
+.messaging-folder-nav-actions {
+  a {
+    color: $color-primary;
+    text-decoration: underline;
+
+    i {
```

Comment:
This doesn't need to be nested inside of the `a` block. `.messaging-folder-nav-actions i` is specific enough.